app/helpers/admin/taggable_content_helper.rb

```diff
@@ -55,6 +55,22 @@ def taggable_ministerial_role_appointments_container
     end
   end
 
+  def taggable_needs_container
+    Rails.cache.fetch("need.linkables", expires_in: 1.minute) do
+      publishing_api_with_low_timeout = Services.publishing_api.dup.tap do |client|
```

Comment:
I'm assuming it's because we want to make needs more robust when it comes to the Publishing API going down, as the same technique is used for policies (https://github.com/alphagov/whitehall/commit/b69d842c4111fcc7cfa39c08bbda623d2fc7262c) but maybe @cbaines can explain this a bit more in detail.